src/components/DayPicker.jsx

```diff
@@ -55,6 +58,9 @@ const defaultProps = {
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
   hidden: false,
+  navPrev: null,
+  navNext: null,
+  useNavDefaultStyles: true,
```

Comment:
Hmm. I totally forgot about those to while implementing this to be honest. I think both of them should have the ability to change this prop.